pytorch_lightning/callbacks/rich_model_summary.py

```diff
@@ -61,7 +61,8 @@ class RichModelSummary(ModelSummary):
     def __init__(self, max_depth: int = 1) -> None:
         if not _RICH_AVAILABLE:
             raise ImportError(
-                "`RichModelSummary` requires `rich` to be installed. Install it by running `pip install rich`."
+                "`RichModelSummary` requires `rich>=10.2.2` to be installed. "
```

Comment:
We have two separate conditions here. Hence, I think we should have two different errors being raised.
One for rich not being installed, other for Rich not being >=10.2.2

Reply:
Will address!